### PR TITLE
send email when request was accepted

### PR DESCRIPTION
**Pull request by NghiaNTT**

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+      <div class="field">
+        <%= f.label :password %>
+        <%= f.password_field :password, autocomplete: "off",
+          class: "form-control" %>
+      </div>
 
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "off" %>
+      <% if devise_mapping.rememberable? -%>
```
Comment (phanbt58) [inline app/views/devise/sessions/new.html.erb]: co cai gi dang truoc %> the a

File: app/views/users/_show.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<% provide :title, current_user.name %>
+<section class="user_info">
+  <% if current_user.avatar.present? %>
+    <%= image_tag(current_user.avatar, size: Settings.users.ava_size) %>
```
Comment (phanbt58) [inline app/views/users/_show.html.erb]: remove ()

File: app/views/users/_show.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  <% else %>
+    <%= image_tag(Settings.users.default_avatar,
```
Comment (phanbt58) [inline app/views/users/_show.html.erb]: remove ()

File: app/views/devise/sessions/new.html.erb [hunk 4]
```diff
@@ -1,26 +1,29 @@
-<h2>Log in</h2>
+<% provide :title, t("views.devise.login") %>
+<h1><%= t "views.devise.login" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
```
Comment (phanbt58) [inline app/views/devise/sessions/new.html.erb]: cho nay remove () có dk k chị

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -1,6 +1,22 @@
...
+        </section>
+      </aside>
+    </div>
+  <% else %>
+    <div class="row">
+      <div class="col-md-8 col-md-offset-3">
+        <h2><%= t "views.static_pages.home.greeting" %></h2>
+      </div>
+      <div class="col-md-8 col-md-offset-5">
+        <%= link_to t("views.static_pages.home.signup_invite"), new_user_registration_path, class: "btn btn-lg btn-primary" %>
```
Comment (phanbt58) [inline app/views/static_pages/home.html.erb]: cai nay k xuong dong nay

**Comment by cuongncFramgia** [comment]
ok

**Pull request merged**